CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -0,0 +1,70 @@
+
+### Changed
+
+- Ensure cni folders are created when starting antrea-agent with containerd on Windows. ([#4685](https://github.com/antrea-io/antrea/pull/4685), [@XinShuYang])
+- Decrease log verbosity value for antrea-agent specified in the Windows manifest for containerd from 4 to 0. ([#4676](https://github.com/antrea-io/antrea/pull/4676), [@XinShuYang])
+- Bump up cni and plugins libraries to v1.1.1. ([#4425](https://github.com/antrea-io/antrea/pull/4425), [@wenyingd])
+- Extend the message length limitation in the Conditions of Antrea-native policies to 256 characters. ([#4574](https://github.com/antrea-io/antrea/pull/4574), [@wenyingd])
+- Stop using ClusterFirstWithHostNet DNSPolicy for antrea-agent; revert it to the default value. ([#4548](https://github.com/antrea-io/antrea/pull/4548), [@antoninbas])
+- Perform Service load balancing within OVS for Multi-cluster Service traffic, when the local member Service of the Multi-cluster Service is selected as the destination. ([#4693](https://github.com/antrea-io/antrea/pull/4693), [@luolanzone])
+- Rename the `multicluster.enable` configuration parameter to `multicluster.enableGateway`. ([#4533](https://github.com/antrea-io/antrea/pull/4533), [@jianjuns])
+- Add the `multicluster.enablePodToPodConnectivity` configuration parameter for antrea-agent to enable Multi-cluster Pod-to-Pod connectivity. ([#4605](https://github.com/antrea-io/antrea/pull/4605), [@hjiajing])
+- No longer install Whereabouts CNI to host. ([#4617](https://github.com/antrea-io/antrea/pull/4617), [@jianjuns])
+- Add an explicit Secret for the `vm-agent` ServiceAccount to the manifest for non-Kubernetes Nodes. ([#4560](https://github.com/antrea-io/antrea/pull/4560), [@wenyingd])
+- Change the `toService.scope` field of Antrea ClusterNetworkPolicy to an enum. ([#4562](https://github.com/antrea-io/antrea/pull/4562), [@GraysonWu])
+
+### Fixed
+
+- Fix route deletion for Service ClusterIP and LoadBalancerIP when AntreaProxy is enabled. ([#4711](https://github.com/antrea-io/antrea/pull/4711), [@tnqn])
+- Fix Service routes being deleted on Agent startup on Windows. ([#4470](https://github.com/antrea-io/antrea/pull/4470), [@hongliangl])
+- Avoid duplicate Node Results in Live Traceflow Status. ([#4715](https://github.com/antrea-io/antrea/pull/4715), [@antoninbas])
+- Fix OpenFlow Group being reused with wrong type because groupDb cache was not cleaned up. ([#4592](https://github.com/antrea-io/antrea/pull/4592), [@ceclinux])
+- Ensure NO_FLOOD is always set for IPsec tunnel ports and TrafficControl ports. ([#4654](https://github.com/antrea-io/antrea/pull/4654) [#4419](https://github.com/antrea-io/antrea/pull/4419), [@xliuxu])
+- Fix Agent crash in dual-stack clusters when any Node is not configured with IP addresses of both address families. ([#4480](https://github.com/antrea-io/antrea/pull/4480), [@hongliangl])
```

Comment:
with an IP address for each address family

Reply:
done